src/trajectory_visualizer.cpp

```diff
@@ -1,7 +1,7 @@
 // Copyright 2022 FastSense, Samsung Research
 #include <memory>
 
-#include "mppic/trajectory_visualizer.hpp"
+#include "mppic/tools/trajectory_visualizer.hpp"
```

Comment:
Something in this PR broke trajectory visualization. The red point for the path integral is is another frame it seems like (?) and the main trajectory selected is no longer highlighted (or buried under the others?) 

When I go back to `develop` things work again.

Reply:
Maybe something to do with the changes to `getOptimizedTrajectory`?

Reply:
Fixed and add parameters for visualization step. So probably you could try visualize huge batches

Reply:
hmm i break it again

Reply:
fixed